### PR TITLE
Add namespace and google for triage party hosting

### PR DESCRIPTION
**Pull request by ameukam**
Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -446,6 +447,18 @@ groups:
...
       - thockin@google.com
 
+  - email-id: k8s-infra-rbac-release-triageparty@kubernetes.io
+    name: k8s-infra-rbac-release-triageparty
+    description: |-
+      ACL for Bug Triage Release Team
+    settings:
+      ReconcileMembers: "true"
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+    members:
```
Comment (ameukam) [inline groups/groups.yaml:457]: @justaugustus @markyjackson-taulia I'll need your help to figure out people that need to be added.

File: infra/gcp/namespaces/ensure-namespaces.sh [hunk 2]
```diff
@@ -148,6 +148,7 @@ ALL_PROJECTS=(
     "k8s-io-prod"
     "k8s-io-canary"
     "perfdash"
+	  "release-triageparty"
```
Comment (bartsmykla) [inline infra/gcp/namespaces/ensure-namespaces.sh:151]: Just a nit: could you align the line?

Comment (ameukam) [reply infra/gcp/namespaces/ensure-namespaces.sh]: Thanks for the catch!

**Comment by dims** [comment]
@bartsmykla will other teams want do host triage party for their own sigs?

**Comment by bartsmykla** [comment]
@dims I actually don't know. I don't know how the tool works and if we specifically need to deploy it for every party interested in it. Will followup with @ameukam

File: groups/groups.yaml [hunk 3]
```diff
@@ -446,6 +447,18 @@ groups:
...
+      - ameukam@gmail.com
+      - bartek@smykla.com
+      - k8s-infra-release-viewers@kubernetes.io
```
[the remark this answers is not in the document]
Comment (justaugustus) [reply groups/groups.yaml:460]: Add the 1.19 Bug Triage Team:
- gianarb92@gmail.com - @gianarb
- jtslear@gmail.com - @jtslear
- droslean@gmail.com - @droslean
- kmbruner@gmail.com - @kbruner
- v@gor.io - @bai

With an inline comment of their role e.g., `# 1.19 Bug Triage Lead`
@gianarb is the Bug Triage Lead, everyone else listed is a shadow.

FYI @kubernetes/release-team-leads -- you're already part of the group as members of `k8s-infra-release-viewers@kubernetes.io`.

File: groups/groups.yaml [hunk 4]
```diff
@@ -446,6 +447,18 @@ groups:
       - stefan.schimanski@gmail.com
       - thockin@google.com
 
+  - email-id: k8s-infra-rbac-release-triageparty@kubernetes.io
```
Comment (justaugustus) [inline groups/groups.yaml:450]: Do we want `release-triageparty` or `triageparty-release` (given that we know that more groups may want to leverage Triage Party)?

File: groups/groups.yaml [hunk 5]
```diff
@@ -446,6 +447,18 @@ groups:
...
+    name: k8s-infra-rbac-release-triageparty
```
Comment (justaugustus) [inline groups/groups.yaml:451]: Same as https://github.com/kubernetes/k8s.io/pull/916#discussion_r434111696.

File: infra/gcp/namespaces/ensure-namespaces.sh [hunk 6]
```diff
@@ -148,6 +148,7 @@ ALL_PROJECTS=(
     "k8s-io-prod"
     "k8s-io-canary"
     "perfdash"
+    "release-triageparty"
```
Comment (justaugustus) [inline infra/gcp/namespaces/ensure-namespaces.sh:151]: Same as https://github.com/kubernetes/k8s.io/pull/916#discussion_r434111696.

File: groups/groups.yaml (hunk 4 above)
Comment (ameukam) [reply groups/groups.yaml:450]: `triageparty-release` is more convenient for the possible scenario you described. Also naming stuff is hard :-).

@dims WDYT ?

**Comment by dims** [comment]
Do you need a DNS entry?

**Comment by ameukam** [comment]
@dims Yes. I wanted to do it in the next PRs.

**Comment by dims** [comment]
/approve

(there's still a hold)

**Comment by justaugustus** [comment]
> Do you need a DNS entry?

@dims -- maybe triage.k8s.io as the subdomain and release.triage.k8s.io as the specific instance?

File: infra/gcp/namespaces/ensure-namespaces.sh [hunk 7]
```diff
@@ -148,6 +148,7 @@ ALL_PROJECTS=(
     "k8s-io-prod"
     "k8s-io-canary"
     "perfdash"
+    "triageparty-release"
     "slack-infra"
     "node-perf-dash"
```
[the remark this answers is not in the document]
Comment (ameukam) [reply infra/gcp/namespaces/ensure-namespaces.sh:153]: I can sort them in this PR but prefer do it in a different one.

[the remark this answers is not in the document]
Comment (justaugustus) [reply infra/gcp/namespaces/ensure-namespaces.sh:153]: Yep, definitely non-blocking.
/hold cancel
/lgtm
/approve

**Pull request merged**